Unit.java

```diff
@@ -1,12 +1,10 @@
-
-
 public class Unit{
-   
-    int atk;
-    int hp;
-    int attackSpeed;
 
-    public Unit(int atk, int hp, int attackSpeed)
+    double atk;
+    double hp;
```

Comment:
ezek maradjanak nyugodtan int-ek, it tnem lenne celszeru kerekitesi dolgokkal foglalkozni.